src/harness/sas_testcase.py

```diff
@@ -100,3 +103,13 @@ def assertRegisteredAndGranted(self, registration_request, grant_request,
 
     # Return cbsd_ids and grant_ids
     return list_of_cbsd_ids_and_grant_ids
+
+  def TriggerDailyActivitiesImmediatelyAndWaitUntilComplete(self):
+    self._sas_admin.TriggerDailyActivitiesImmediately()
+    signal.signal(signal.SIGALRM, timeout_error)
```

Comment:
It might be easier to read the code if you just used a lambda here to raise the exception.